### DISCUSSION TITLE
Who wants to be hired (June 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#37380] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #36737. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #37379.

**Comment by stefkudla** [comment]
I'm Stefan from Las Vegas, Nevada. I'm passionate about Jamstack and headless architecture, and am looking for my first position as an engineer to provide value and grow/teach alongside others!

Location: Las Vegas, Nevada, USA
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, React, Next, Tailwind, Node, Cosmicjs, GraphCMS, Framer Motion, HTML, CSS, Firebase, Figma, Git, Vercel, Netlify
Resume/CV: [Stefan Kudla's Resume](https://stefankudla.com/Stefan_Kudla_Resume.pdf)
Website: https://stefankudla.com
Email: stefan@stefankudla.com

**Reply by RayDev1988** [reply]
Senior JavaScript developer who specializes in React and Vue for front end and Node.JS for back end.
It will be great if we can make schedule and discuss further detail
Raymond

**Reply by ghost** [reply]
As a full stack developer who specializes in React, Vue, Express for front end and Node.JS, PHP, Django for back end.
It will be great if we can make schedule and discuss further detail
James Donaldo

**Reply by ghost** [reply]
jd3606192@gmail.com

**Reply by spencerbart** [reply]
Love your website!

**Reply by stefkudla** [reply]
Thanks!

**Comment by simonhlee97** [comment]
Hi. I'm Simon, from Indianapolis, IN, but currently in South Korea. I'm looking for remote and independent-contractor opportunities in a front-end role, working with React, Next.js, NodeJS, MongoDB, Tailwind, Styled Components, Bootstrap, etc.

Location: Incheon, Korea
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, React, Next, Tailwind, Node, HTML, CSS, Firebase, Git, Vercel, Netlify
Resume/CV: [My Resume](https://github.com/simonhlee97/career-docs/blob/main/resume.simon.lee.pdf)
Website: https://simonhlee97.github.io/
Email: [simonhlee4945@gmail.com](mailto:simonhlee4945@gmail.com)

**Reply by ghost** [reply]
As a full stack developer who specializes in React, Vue, Express for front end and Node.JS, PHP, Django for back end.
It will be great if we can make schedule and discuss further detail
James Donaldo

jd3606192@gmail.com

**Comment by aboubakarch** [comment]
Hi Everyone,

This is Abou Bakar. I am a passionate frontend developer specializing in various aspects of frontend development and looking for new opportunities. I've more than 3.5 years of experience to build multifunctional web applications for various clients and companies. I've very strong experience in React,NextJS, Javascript/Typescript, TailwindCSS,SCSS,Styled components. I am currently looking for a new career opportunity. I'm very committed to my work and always give my 101% to deliver the best solutions.
I'm looking forward to hearing anyone of you.
Best Regards.

Location: Lahore, Pakistan
Remote: Yes
Willing to relocate: Yes
Technologies: NextJS, ReactJs, React Native, React Native Web, Redux/Persist-Redux/Redux-ToolKit, TailwindCSS, MaterialUI, AntDesign, SCSS, Strapi, Directus, PostgresSQL, MongoDB, Djangorestframework, Typescript, firebase, Stripe, GCP/Heroku/Netlify/Digital Ocean, GitHub/BitBucket/GitLabs, Jira/Trello/ActiveCollabs, currently learning blockchain.
Resume: [Linkedin](https://www.linkedin.com/in/iamaboubakar/) OR [Resume](https://drive.google.com/file/d/1UMQ-E0ZePPJEFCaPpLREi_6C0z-mH5y_/view?usp=sharing)
Email: [iamaboubakkar@gmail.com](mailto:iamaboubakkar@gmail.com)

**Reply by ghost** [reply]
As a full stack developer who specializes in React, Vue, Express for front end and Node.JS, PHP, Django for back end.
It will be great if we can make schedule and discuss further detail
James Donaldo

jd3606192@gmail.com

**Comment by Koreankat** [comment]
Hello everyone i'm Ali , junior front end developer.
Location: Casablanca, Morocco
Remote: Yes
Willing to relocate: Yes
Technologies:  React, Tailwind, Node, Javascript ,HTML, CSS, Figma, Git, Vercel, 
Resume/CV: https://resume.io/r/UhC8zWZ9y
Website:https://ali-portfolio-dun.vercel.app/
Email: ali007jaafari@gmail.com

**Comment by misqke** [comment]
Hi, I'm Mike, a front-end developer.  I focus towards react development and Next.js is my framework of choice.  I also have experience with some back-end technologies and have built a few full -stack projects.  I am looking for local or remote work (Sorry, I can't relocate at this time).

Location: Ellwood City, PA,
Remote: true,
Willing to relocate: false,
Technologies: React, Next, Redux, SCSS, Node, Express, MongoDB, Puppeteer, Git, 
Resume: [my resume](https://www.mikerustportfolio.com/resume),
Website: [my portfolio](https://www.mikerustportfolio.com),
Email: misqke.rust@yahoo.com

**Comment by RieserStern** [comment]
Hi, I am a Full Stack developer, who has 7 years of strong experience developing. Over the last 3 years, mainly focusing on blockchain technology. As a Full stack developer, have enough experience developing with React, Angular, Node, Vue, TypeScript, Python, JavaScript, PHP, HTML/CSS, Wordpress, Yii2, Laravel etc
Even also have experience developing Smart Contract with Solidity, Solana, Rust, Web3, Solana-web3. And have experience developing Front and Back-end for 6+ years and proficient in Git.
Recently, I am working as a Front-end developer for integration the Website with Smart Contract. And I have been developing lots of programming languages, frameworks, libraries, platforms.

Location: Oshawa, ON, Canada,
Remote: true,
Willing to relocate: false,
Technologies: React, Next, Redux, JavaScript, TypeScript, SCSS, Node, Express, Vue, Angular, MongoDB, MySQL, Git,
Resume: [my resume](https://cvdesignr.com/p/61dd927e987ef),
Email: RieserStern@gmail.com

**Comment by kellenwiltshire** [comment]
Hi Everyone,

My name is Kellen Wiltshire and I am a passionate full-stack developer specializing in all aspects of Web Development looking for the next step in my journey. I have the experience to create beautiful and functional websites combining the latest frameworks and technologies. I have experience in React(NextJS), Javascript/Typescript, TailwindCSS, Styled Components, as well as traditional HTML5 and CSS3. I am currently look for a new Web Development career opportunity! If you are willing to give me the opportunity I promise you won't find someone more motivated to exceed expectations than me!

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify, Currently learning Vue
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by Phoenix-Alpha** [comment]
Hi,

I am Alex, result-driven software engineer with over 8 years of professional career in startups and looking for full time position.

Location: Bulgaria
Remote: Yes
Willing to relocate: No
Technologies: React, Next, Tailwind CSS, GQL, Hasura, Angular, NgRx, RxJS, AWS Cloud & DevOps
Resume or Website: https://talent.hubstaff.com/profiles/alex-jin-2
Email: alexjin0704@gmail.com

Looking forward to talking to you!

Regards

**Comment by sabinbaniya** [comment]
Location: Nepal
Remote: yes
Willing to relocate: no
Technologies: React.JS ,Next.JS, Node.JS, Express.JS ,git, Tailwind CSS, Material UI
Resume/CV: https://portfolio.sabinbaniya.com.np/assets/sabinbaniya.pdf
Website: https://portfolio.sabinbaniya.com.np/
Email: baniya.sabinn@gmail.com

**Comment by mgorabbani** [comment]
Hi, I'm Golam, a front-end developer. I focus towards react and typescript. I also have experience with some back-end technologies and have built a few full -stack projects. I am looking for remote work. 

Location: Berlin, Germany
Remote: true,
Willing to relocate: false,
Technologies: React, Next, Typescript, Javascript, Redux, SCSS AntDesign, MUI, Docker, AWS, GCP
Resume: [my resume](https://drive.google.com/file/d/1HsnsnXI8HomV3Eu9xcXiBNi8b9kNMcxw/view?usp=sharing),
Website: [my portfolio](https://rabbani.dev/),
Email: [mgorabbani@gmail.com](mailto:mgorabbani@gmail.com)

**Comment by biswarup35** [comment]
Hi :wave:
I am Biswarup, I have been doing front-end development with Next.js and React.js for almost two years now. Currently, I am building a React UI library based on Google's material 3 design system. I am also looking for a remote junior-level front-end developer role.

Location: West Bengal, India
Remote: yes
Willing to relocate: no
Technologies: React.js, Next.js, TypeScript, JavaScript, Node.js, Express.js, Emotion,js, Git and GitHub.
Resume: [my resume](https://drive.google.com/file/d/1bKR2flDNlEJOIcst1VylnNHFtI7Vkp01/view?usp=sharing)
Website: [my website](https://biswarup.vercel.app/)
Email: <biswarup35@gmail.com>

**Comment by Dreamfullstacker** [comment]
Hi Clients
I’m full stack web developer and senior blockchain developer.
I have 5years experience on MERN stack developement and 7years experience on WAMP, LAMP development.
Laravel+ React or Larave + Vue.js is my special field
I also have rich experience with  Blockchain and AWS.
Timezone is no problem for me.
Passionate, driven and goal oriented are the best adjectives to describe me.
I like to be in the back and front of tech hence my passion for web and blockchain development. ⏰
I put my heart and soul 🧠into every project I work on✍️, going non-stop till it is brought to perfection.📈
====Blockchain====
 ✨Chain : Bitcoin, Ethereum, Solana, BSC, Polygon, Avalanche, Tron 
✨Cryptocurrency : ERC-20, ERC-721, ERC-1155, ERC-777, ERC-223, BEP-2, BEP-20, BEP-721, TRC-20 
✨Wallet Connect : Meta Mask, Trust Wallet, Coinbase and more using Web3.js 
✨NFT : NFT minting, Staking, marketplace, landing page, Gamesite
✨Token : Swap, bridge, staking. 
✨Trading View 
====Web Dev & Design==== 
✨PHP framework : CI, Laravel, Word press 
✨Java script : jQuery, React.js(Next.JS, Gatsby), Vue.js(Nuxt.JS), AngularJS, Svelte 
✨Database : MySQL, MongoDB, RDS, AWS DynamoDB, Firebase
✨Frontend Library : Bootstrap, Tailwind CSS, MUI, React Kit, Ant D 
✨Other : HTML5, Figma , Shopify , Three.js
====Amazon Web Service====
✨AWS S3
✨AWS Dynamo DB
✨AWS Lambda
✨AWS IVS
✨AWS API Gateway
✨AWS RDS
✨AWS Web socket
✨AWS Amplify
====Mobile Dev & Design====
✨React Native 
✨Flutter 
✨Swift
====Odoo + Vue storefront====
I guarantee that you will get more than you've expected.
If you hire me, I will do my best for you 
I know I may not have all the accolades and experience you're looking for, but trust me, I have the creative mind, passion and commitment every recruiter or company is looking for.
Contact me for any type of work to be done right and perfectly.
Thank you for your reading.
Send DM to me please.
Best regards
Email: dreamfullstacker@gmail.com

**Comment by sntwandy** [comment]
Hi 👋, I'm Wandy Santana, I am a developer passionate about technology and its development, especially JavaScript technologies such as React.js and Next.js, my favorite framework to work and develop those versatile graphical interfaces.

Location: the Dominican Republic 🇩🇴.
Remote: Yes. 😀
Willing to relocate: If they offer me the opportunity, yes. 😄
Technologies: JavaScript, TypeScript, React.js, Next.js, Node.js, TalwindCss, Material-UI, Git and GitHub, Linux, AWS, Vercel.
Resume/CV: [https://www.linkedin.com/in/sntwandy/](https://www.linkedin.com/in/sntwandy/)
Website: [https://github.com/sntwandy](https://github.com/sntwandy)
(My website is still in progress)
Email: sntwandy@gmail.com

**Comment by VianaArthur** [comment]
Hi. I'm Arthur Viana, from Belo Horizonte - Brazil 😄 

I'm looking for remote opportunities in a front-end role, working with React, Next.js, NodeJS, Tailwind

Location: Belo Horizonte, Brazil
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, TypeScript, React JS, Next.js, Tailwind, Node JS, HTML, CSS, Firebase, SQL, Git, Vercel
Resume/CV: [My Resume](https://www.datocms-assets.com/57819/1654564685-arthurviana_cv.pdf)
Website: https://arthurviana.com/
Email: [arthur.oliveiraviana@gmail.com](mailto:arthur.oliveiraviana@gmail.com)

**Reply by Umair-Ramees** [reply]
HI

**Comment by Infosourcer** [comment]
Hi John,

Thanks for the note and info!

Would you do me a favor and reply with a resume and l will forward to the
hiring manager as I am actually traveling for the rest of the month.

Once the team gives me a green light, I can get you started with intros and
interviews.

Does that sound good to you?

Thank you!
Suzy Tonini
Program Manager/Innovation Accelerator
Cloudera


On Wed, Jun 8, 2022 at 8:38 AM bluespider ***@***.***> wrote:

> Hello. @Infosourcer <https://github.com/Infosourcer>.
> I reviewed your job post description on the discord social network and was
> interested in your post.
>
> I am John. I am an ambitious Senior Front end Developer who loves working
> on web development.
> and also I have some experience in blockchain development such as building
> NFTs minting site and NFTs market place.
> I have a great experience in front end development as well as backend
> development.
> I am looking forward to getting an opportunity to grow my client.
>
> Technologies:
> ======== front end development ========
>
> React.js/Next.js/Gatsby/GraphQL
> Vue.js/Nuxt.js
> TypeScript/ES6/ES7/ES8
> Vanilla JS/JQuery
> ======== back end development ========
>
> Node.js/Express.js/Nest.js
> PHP/Laravel
> ======== blockchain development ========
>
> Web.js, Ether.js, Alchemy, Remix, Truffle, Hardhat, Moralis
> Solidity, ERC20/ERC721/ERC1155
> -Ethereum, Solana, Polygon, etc
> ======== devOps development ========
>
> AWS cloud services
> Firebase
> Vercel/Heroku
> CI/CD
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/next.js/discussions/37380#discussioncomment-2906276>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAM7MMRFLGO3NY64FJEVSCTVOC5AHANCNFSM5XSX5DQA>
> .
> You are receiving this because you were mentioned.Message ID: <vercel/next
> .***@***.***>
>
-- 
Thanks,

Suzy Tonini
***@***.***
360-201-4285

**Comment by ghost** [comment]
You are just flexible.

**Comment by ghost** [comment]
I am not john.
I am called James.

**Comment by ghost** [comment]
I am james Donaldo
My skill sets:
React Redux ReactNative Nextjs
Vuejs Rxjs Anglar.js
PostgreSql Mongo MySql
Node.js Laravel Python Django PHP
AWS GoogleCloud Azure Firebase Heroku

please contact me

**Comment by BrimmedHat** [comment]
My name's Joshua, I love making apps and sites with Next!

Location: Toronto, Canada
Remote: Preferable
Willing to relocate: Yes
Technologies: react, next, node, express, mongoDB
Website: https://joneilltechnicalportfolio.vercel.app/
Resume/CV: https://resume.io/r/srFhKDi7i
Email: joneilltechnical@gmail.com

**Comment by abayomi1196** [comment]
Hi, I'm Ayo from Abuja, Nigeria. I'm passionate about Javascript and Frontend web development and would love an opportunity to contribute to building performant world-class applications that contribute to making the world a better place.

Location: Abuja, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, React, Vue, Next, TailwindCSS, Node,  Express, HTML, CSS, Firebase, Figma, Git, Vercel, Netlify.
Resume/CV: [Ayo's Resume / LinkedIn Profile](https://www.linkedin.com/in/abayomi-ayodele/)
Website: https://portfolio-ayodele.vercel.app/
Email: [ayodele@duck.com](mailto:ayodele@duck.com)

**Comment by Soft1109** [comment]
hi, Nice to meet you 
How are you?
I am a senior smart contract & Next.js & React.js expert.
https://mooningmonkey.com
https://wulfznft.com
plz, check my previous works.
https://miracle1109.github.io 
It is my porfolio.
Looking forward to hearing from client.

**Comment by idirmosh** [comment]
Somewhat full-stack developer looking for work opportunity, I am somewhat very comfortable with the react eco-system with a passion of learning new things.

**Location**: Greenacres, Florida, US
**Remote**: Yes
**Willing to relocate**: NO
**Technologies**: React. Nextjs. Node. Express. GraphQl, JavaScript. all css-in-js tech. 
**Design**: I have a good eye for user interfaces and web design in general.
**Résumé/CV**: I am currently working on the cv just moved to us not too long ago, currently working on getting the GED.
**Projects**: currently working on an issue-tracker app, the source can be found on my github :https://github.com/idirmosh/issuestracker, its still work on progress, hopefully it would be a demo of my coding skills.
**Website**: idirmosh.dev
**Email**:  idir.mosh@gmail.com
**LinkedIn**: https://www.linkedin.com/in/idirmosh/

Hit me up for any remote work, I would appreciate any opportunity.

**Comment by biratdevpoudel** [comment]
Location: Nepal
Remote: Yes
Willing to relocate: No
CV: I am currently studying Software Engineering at Gandaki College of Engineering and Science.
Technologies: React.JS, Material UI ,Next.JS, Python Django, PostgreSql , Git, Tailwind CSS, Vercel, Netlify, Figma.
Website: https://biratdevpoudel.com.np
Email: [biratpoudel02@gmail.com](mailto:biratpoudel02@gmail.com)

**Reply by satishchandravrvs** [reply]
Location: Hyderabad, India
Remote: Yes
Willing to relocate: Yes
.
Technologies: Azure DevOps, Azure Cloud, System administration, Desktop Support.
Technical Support, Azure Pipelines, Github, Azure Repos, IaC( ARM Templates)
Website: www.linkedin.com/in/satish-chandra-vrvs-7ba0a975 
Resume: https://resume.io/r/gLOQUVvPL
Email: satishchandra.vrvs@gmail.com

**Comment by ghost** [comment]
Please describe your more detail about tech.
Thanks.
James.

**Comment by mahdisoultana** [comment]
Hello Everyone ,i'm Mahdi React developer and i love working client projects with Nextjs
Location:Morocco
Remote:YES
Willing to relocate:YES
Technologies: ReactJS,NextJS,graphql,react-query,react-flow,socket.io,nodeJS,monogDB,webRTC(PeerJS),CSS,HTML,JS
Resume/CV:[Mahdi-CV.jpg](https://drive.google.com/file/d/1rGp9NwlUHDhxo2xsZhvbPo7osKCrQfai/view?usp=sharing)
Website:https://github.com/Mahdi-Soultana
Email:mahdisoultana@gmail.com

Thank you for the opportunity !

**Comment by syedsimanta03** [comment]
Front-end coder & UI/UX designer 
============================
Location:  Bangladesh 
Remote: Yes
Willing to relocate: Yes
Technologies: React, Nextjs, Vue, Nuxt, Svelte, Svelte kit, tailwind, Bootstrap, HTML, CSS, SASS, WordPress, NFT minting site
Resume/CV: https://designcoder.netlify.app/cv
Website: https://designcoder.netlify.app
Email: syed.simanta10@gmail.com

**Comment by aka-user15** [comment]
I am Azam
I am a full stack web developer I have been coding for 4 years and I have made many web apps using Next JS and I want to work remotely.


Website: https://azamkhan.vercel.app
Email: azam0417@gmail.com

**Comment by ghost** [comment]
Hello, everyone. I’m a **senior full-stack engineer** with 10+ years of experience in web development, managing all aspects of the development process. I've worked on 70+ projects and solved complex business problems in various industries including marketing, healthcare, eCommerce, real estate, and car insurance. I have consistently saved costs while increasing profits. I am ready for an opportunity to use my skills to grow your revenue exponentially. 

**Location**: Canada
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: 
- languages
   * JS, TS, PHP, Python
- front-end
   * React, Next.js, Vue.js, Nuxt.js, Angular for JS frameworks/libraries
   * Bootstrap, Material UI, Tailwind CSS, Vuetify for UI frameworks
- back-end
  * Express, Laravel for frameworks
  * MySQL, MongoDB, PostgreSQL for database
- services && libraries
  * Mocha, Enzyme, Chai for testing
  * Github, Bitbucket for version control
  * AWS, Netlify, Heroku, Firebase for deployment

**Resume**: [linkedin](https://www.linkedin.com/in/lucas-jiang-486516244/)
**Website**: https://github.com/star-dev-jiang
**Email**: [lucas.jiang.itdev@gmail.com](https://lucas.jiang.itdev@gmail.com)

My goal has always been to exceed the expectations of my clients.
If you have a challenging or interesting project, don't hesitate to contact me.
Thank you for your careful attention.

Best regards.🎉

**Comment by Oxicode** [comment]
I am a full-stack developer focused on WS integrations with Artificial Intelligence 🤖 from the cloud. I have solid experience in technologies like Javascript/NodeJS (8 years), PHP (11 years), Python (3 years), also other technologies like Sharepoint and React.
I hold some prestigious certifications in the industry from Microsoft such as Azure Architect Design and Designing and Implementing an AI Solution and others.

I believe that my talent has added a lot of value to the technical teams where I have participated because, in the years of analysis, development, implementation and leadership (3 years) in software projects I have demonstrated that products are born and evolve over time, done before perfect.

Finally, I am very passionate about learning new technologies and techniques, finding solutions to challenges whether they are work related, student related or from the same community in Github. One of this topic has been Artificial Intelligence.

Location: Lima, Perú
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, React, Next, Tailwind, Node, Firebase, Figma, Git, Aws, Azure
Resume/CV: [Oxicode Resume](https://oxicode.io/pdf)
Website: https://oxicode.io
Email: [christian.quispeh@gmail.com](mailto:christian.quispeh@gmail.com)

**Comment by CodeMan-github** [comment]
Location: Rowley Regis, west midlands, United Kingdom
Remote: Yes
Willing to relocate: No
Technologies: Typescript, Next, Nest, React, Node.js, Prisma, Git, MySQL
Resume/CV: [https://www.linkedin.com/in/eddie-long-00ba61245/](url)
Website: [https://www.linkedin.com/in/eddie-long-00ba61245/](url)
Email: webdev1988@outlook.com

**Comment by TylerAHolden** [comment]
Hello I'm Tyler

I have over 12+ years experience in web development, managing/growing dev teams, starting businesses, product management, UI/UX design, data analysis and customer relations. I am passionate about beautiful UI/UX and scaling teams.  

**Location**: Corpus Christi, TX
**Remote**: Yes
**Willing to relocate**: No probably not
**Technologies**:
<p>
    <a href="https://github.com/search?q=user%3TylerAHolden+language%3AtypeScript"><img alt="TypeScript" src="https://img.shields.io/badge/TypeScript-007ACC.svg?logo=typescript&logoColor=white"></a>  
    <a href="https://github.com/search?q=user%3TylerAHolden+language%3Ajavascript"><img alt="Node.js" src="https://img.shields.io/badge/Node.js-43853D.svg?logo=node.js&logoColor=white"></a>
    <a href="https://github.com/search?q=user%3TylerAHolden+language%3Asql"><img alt="SQL" src="https://custom-icon-badges.herokuapp.com/badge/SQL-025E8C.svg?logo=database&logoColor=white"></a>
<a href="#"><img alt="React" src="https://img.shields.io/badge/React-20232a.svg?logo=react&logoColor=%2361DAFB"></a>
<a href="#"><img alt="Ionic" src="https://img.shields.io/badge/Ionic-3880FF?logo=ionic&logoColor=white"></a>
<a href="#"><img alt="NextJS" src="https://img.shields.io/badge/next.js-000000?logo=nextdotjs&logoColor=white"></a>
<a href="#"><img alt="Firebase" src="https://img.shields.io/badge/firebase-ffca28?logo=firebase&logoColor=white"></a>
<a href="#"><img alt="React Native" src="https://img.shields.io/badge/React_Native-20232A?logo=react&logoColor=61DAFB"></a>
<a href="#"><img alt="Firebase" src="https://img.shields.io/badge/firebase-ffca28?logo=firebase&logoColor=white"></a>
<a href="#"><img alt="MongoDB" src ="https://img.shields.io/badge/MongoDB-4ea94b.svg?logo=mongodb&logoColor=white"></a>
<a href="#"><img alt="GraphQL" src ="https://img.shields.io/badge/GraphQl-E10098?logo=graphql&logoColor=white"></a>
    ... and many more... Check my GitHub profile for a long list!
</p>

**Resume/CV**: https://resume.io/r/ISzmfpQX8
**Website**: https://www.linkedin.com/in/tyler-holden-b7750a129/
**Email**: tyleraholden@gmail.com

**Comment by scheung38** [comment]
Hi My name is Sebastian Cheung, MSc, CQF, 20+ years combined background in DevOps, Full-Stack, mobile and silicon engineering. Contracting only roles.

Looking for interesting startups making an impact along the way

Technologies: Django, Kubernetes, Azure/AWS/GCP, Docker, Kafka, RedPanda, Web, Flutter, React, Hasura, 

Location: London, UK
Remote: Yes
Willing to relocate: No
Resume or Website: https//github.com/scheung38/, https://www.linkedin.com/in/scheungcqf/
Email: [sebastian_cheung@live.com](mailto:sebastian_cheung@live.com)

Looking forward to hearing from you!

Best Regards

**Comment by hasan4it** [comment]
Hi
I am Hasan Baig a full stack developer with small set of technolgiesvlike html, css, javascript, PHP, databases like mysql and postregress sql. I can work on custom php and frameworks of php like Laravel and Symfony. I can also work with cms like WordPress and winter. On server side i can work on Ubuntu based EC2, LAMP stack servers. I can configure apache and nignx servers for PHP projects deployments. Security is an essenial part of my deployments. Beside all this i can optimize the websites for SEO and page speed. In AWS services i am confortable to work on EC2, RDS and S3, and know how of SQS and SNS services. In front-end technologies i can work with custom responsive flex based css and its frameworks liks bootstrap and tailwind. 

Regards 
Muhammad Hasan Baig 
Heyhasan.com
